pkg/volume/fc/fc_util.go

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fc
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/volume"
+)
+
+// given a disk path like /dev/sdx, find the devicemapper parent
```

Comment:
Sorry, I might be missing something, but I don't see which tests in fc_test.go add coverage for this code, since those tests use a `fakeMounter`.

The only reason I'd like to see more coverage here is because this code is doing a lot of things each of which could potentially fail.